tools/release/__init__.py

```diff
@@ -105,13 +105,8 @@ def update_version_file(version: str, filepath: str) -> None:
 
 
 def create_git_tag(session: Session, tag_name: str, *, message: str) -> None:
-    session.run(
-        # fmt: off
-        "git", "tag", "-m", message, tag_name,
-        # fmt: on
-        external=True,
-        silent=True,
-    )
+    cmd = ["git", "tag", "-m", message, tag_name]
+    session.run(*cmd, external=True, silent=True)
```

Comment:
Manual change, unrelated to ruff? I can accept that the changed version is better, but IMO we shouldn't be mixing changes to the automation and manual reformatting in the one PR.

Reply:
Yes I should have done that in another commit.